lake/data/writer.go

```diff
@@ -26,7 +26,7 @@ type Writer struct {
 	seekIndexStride  int
 	seekIndexTrigger int
 	first            bool
-	from             *zed.Value
+	seek_min         *zed.Value
```

Comment:
Nit: Ditch the underscore.
```suggestion
	seekMin         *zed.Value
```